includes/admin/class-wcmp-export.php

```diff
@@ -1595,14 +1596,25 @@ public function generateCustomsDeclaration(WC_Order $wcOrder): CustomsDeclaratio
             ->setWeight($totalWeight);
 
         foreach ($wcOrder->get_items() as $item) {
-            $product       = $item->get_product();
-            $productHelper = new ExportRow($wcOrder, $product);
+            $product = $item->get_product();
```

Comment:
dit hele blokje binnen de foreach kan je in een functie trekken

Reply:
...hoewel dat misschien nog niet zo nice is omdat dit bestand al zo groot is, misschien na een grondige refactor ooit op een dag

Reply:
Ja het kan idd maar de niceheid viel mij ook tegen toen ik t deed, dus toch zo gelaten.